### DISCUSSION TITLE
Buying consultation, tablet with linux

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [tycrek/degoogle#474] [Q&A]
Hello everyone, 
since 3 years, that I switched to linux, and got familiar with open source world, can not go back to the jailed propriety privacy problematic world again.
Thanks for providing such a good repo so we can see and comparison degoogled products.

I am in search for a tablet, to do my university jobs, it should have **pen** and also **lte** internet support.
I prefer linux tablet, but there are not any (at least in German market where I am located.)
JingPad is dead.

I'm thinking of HP chromebooks and install linux on it.
Anyone has ideas and can guide me where to get consultation?
Thanks in advance

**Comment by Technetium1** [comment]
Get used online, not Chromebook, the bootloader is shit.

**Comment by tycrek** [comment]
Sorry for the necro bump but I'd advise against any chromebook as well. It has quite poor linux support, and the hardware is generally underpowered unless you pay way too much for what it's worth (and the same amount gets a better "typical" laptop/tablet combo).

My two-cents is a Lenovo Flex. They typically all come with the Lenovo Digital Pen (verify with your vendors) and are a 2-in-1 laptop (flips from laptop to tablet and back). It's what I've been using for about 4 years through the pandemic and beyond, using Arch linux with very little problems aside from post-install hiccups on my part. I can't personally speak on how well it functions as a tablet as I just use it in laptop mode.

**Reply by Shahin-rmz** [reply]
2 in 1 laptops aren't my favorits either.
But I also got one.
meh...
